README.md

```diff
@@ -10,32 +10,8 @@
 - HTML
 - CSS
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+## Demo 
+Feel free to visit the site here https://michaelasong.github.io/PortfolioProject/
```

Comment:
- [ ] In your `README` file, there is no "Getting Started" section. Kindly add the getting started section. This should be an example of how you may give instructions on setting up your project locally. :+1: